### ISSUE TITLE
Support for other trackers

### ISSUE THREAD

**Issue by vikstrous** [vikstrous/pirate-get#50]
Since we've had issues with TPB going down many times, we should make pirate-get a bit more modular and add support for more trackers. Here's the list of popular trackers that we should support:

http://torrentfreak.com/top-popular-torrent-sites-2015-150104/

**Comment by vikstrous** [comment]
While we are at it, we should make sure that we take advantage of trackers that support downloading torrents files directly instead of using the torrent caching service for all trackers.

**Comment by mgaulton** [comment]
I'm be curious to see this added. Any thoughts on how to make that happen?

**Comment by rnhmjoj** [comment]
Every website would require an ad-hoc parser or an API. It would be a lot of work to do and a major rewrite of pirate-get. I think a better approach is to write a simple search engine for the bittorrent DHT. This way it will be really raid-proof.

**Comment by vikstrous** [comment]
This? http://www.digbt.org/

**Comment by rnhmjoj** [comment]
Yes, something like that. The problem is I have no idea how these crawlers work.

**Comment by vikstrous** [comment]
I figured out the details at one point. I don't remember exactly how it works any more, but it basically runs multiple peers on the DHT and for every search someone makes, it records the hash of the torrent. Then it makes the same search itself, downloads to torrent file and indexes it.
